### PR TITLE
ui: misc. v2 db pages fixes

### PR DESCRIPTION
**Pull request by xinhaoz**
- Provide column sorting fns for columns which are transformed from their raw values.
- Fix table empty placeholder image.
- Use `items` prop in antd Tabs instead of TabPane

Epic: CRDB-37558
Release note: None

### CODE REVIEW

File: pkg/ui/workspaces/cluster-ui/src/databasesV2/utils.ts [hunk 1]
```diff
@@ -36,9 +36,9 @@ export const rawDatabaseMetadataToDatabaseRows = (
     return {
       name: db.db_name,
       id: db.db_id,
-      tableCount: db.table_count,
-      approximateDiskSizeBytes: db.size_bytes,
-      rangeCount: db.table_count,
+      tableCount: db.table_count ?? 0,
```
Comment (dhartunian) [inline pkg/ui/workspaces/cluster-ui/src/databasesV2/utils.ts:39]: is there any way to avoid having to do stuff like this? When are these undefined?

Comment (xinhaoz) [reply pkg/ui/workspaces/cluster-ui/src/databasesV2/utils.ts:39]: It's for cases where there are no tables in the DB. We could also make sure the API always returns numbers.

Comment (xinhaoz) [reply pkg/ui/workspaces/cluster-ui/src/databasesV2/utils.ts:39]: We can certainly add that on the API - even so I don't think removign this check hurts or should block this PR.
cc @kyle-a-wong Do you want to pick up that task?

Comment (kyle-a-wong) [reply pkg/ui/workspaces/cluster-ui/src/databasesV2/utils.ts:39]: Oh oops, Yeah these shouldn't have `omitempty` in the struct definition. I think i had goland auto generate these tags and i forgot to remove it. I agree though ,these shouldn't be undefined and should always exist.

fix PR: https://github.com/cockroachdb/cockroach/pull/131441

**Review by kyle-a-wong — APPROVED** [review]

**Comment by xinhaoz** [comment]
Will remove the null checks in one of the open follow-up UI PRs, tftr!
bors r+

**Pull request merged**